### PR TITLE
Implement styled-components in kotlin

### PR DESCRIPTION
**Pull request by tretikoff**
This PR add styled-components implementation in kotlin. I've tried to save most of the interfaces and the main flow.

One of the main differences from styled-components is missing prefixer in this implementation

### CODE REVIEW

File: kotlin-styled-next/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+description=Kotlin implementation of styled-components
+author=Konstantin Tretiakov
```
Comment (tretikoff) [inline kotlin-styled-next/gradle.properties:2]: Не уверен кого в авторах писать здесь

File: kotlin-styled-next/README.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
+[![Bintray](https://img.shields.io/bintray/v/kotlin/kotlin-js-wrappers/kotlin-styled)](https://bintray.com/kotlin/kotlin-js-wrappers/kotlin-styled)
```
Comment (tretikoff) [inline kotlin-styled-next/README.md:1]: README пока скопирован с kotlin-styled

**Review by Leonya — CHANGES_REQUESTED** [review]

**Comment by Leonya** [comment]
Пойду релизить!

**Comment by Leonya** [comment]
Released in pre.220

**Pull request merged**